batch/build-batch-worker-image-startup.sh

```diff
@@ -40,7 +40,7 @@ curl -fsSL "https://github.com/GoogleCloudPlatform/docker-credential-gcr/release
 docker-credential-gcr configure-docker
 
 docker pull ubuntu:18.04
-docker pull google/cloud-sdk:269.0.0-alpine
+docker pull gcr.io/google.com/cloudsdktool/cloud-sdk:310.0.0-alpine
```

Comment:
Let's make a new disk image and update the instance creation code.

Reply:
OK, good, I also want to fold in some stuff from isntance_pool.py in here (like the daemon.json) to try and speed up worker start up.

I have to work on hail team and ATGU meeting prep, but will return to this next week.